### DISCUSSION TITLE
Running Reports

### DISCUSSION THREAD

**Discussion by KernOil** [Budibase/budibase#1250] [Help]
Hi is there a way I can out put a report on request?

I would love to have a automation that after new data is inputted it will then generate a report and email the report to the correct people.  

Any help / work around would be great!
Thank you

**Comment by joebudi** [comment]
If a table/data is updated, you can trigger an email automation with the updated data. You cannot generate a report. Once again, pdf generation is in our roadmap.

Thanks for raising these discussions. It's great to understand the features users need.

**Reply by KernOil** [reply]
thank you, I appreciate your response. I look forward to the coming updates. keep up the good work. I think you have something really good your building here!

**Reply by joebudi** [reply]
Your words mean a lot. Thank you ❤️
